app/controllers/searches_controller.rb

```diff
@@ -195,7 +195,7 @@ def replied
   def favoriting
     @user_items = build_user_items(@searched_tw_user.favoriting)
     @graph = @searched_tw_user.favoriting_graph
-    @tweet_text = view_context.close_friends_text(@user_items.slice(0, 3).map{|i| i[:target] }, @searched_tw_user)
+    @tweet_text = view_context.close_friends_text(@user_items.slice(0, 3).map { |i| i[:target] }, @searched_tw_user)
```

Comment:
__[rubocop]__
* Line is too long. [83/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)